CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## [Unreleased]
 
+- update jruby-utils to 3.2.2, which brings in JRuby 9.2.17.0
```

Comment:
I'm thinking we should probably do a release of this sooner rather than later, so we can get it integrating. unless there are other bumps you had planned?

Reply:
Nope, just wasn't urgent so I didn't. But I can.

Reply:
Updated